crates/apollo-compiler/test_data/diagnostics/0088_fragment_selection_set.graphql

```diff
@@ -13,6 +13,9 @@ query getProduct {
     topProduct {
         ...productFragment
     }
+    topProducts {
+        ...productFragment
+    }
```

Comment:
the fragment won't try to be validated if we don't call it, so calling it again inside a correct field